packages/create-gatsby/src/init-starter.ts

```diff
@@ -68,21 +76,41 @@ const createInitialGitCommit = async (rootPath: string): Promise<void> => {
   }
 }
 
+const setNameInPackage = async (
+  sitePath: string,
+  name: string
+): Promise<void> => {
+  const packageJsonPath = path.join(sitePath, `package.json`)
+  const packageJson = await fs.readJSON(packageJsonPath)
+  packageJson.name = kebabify(name)
+  packageJson.description = `My Gatsby site`
+  try {
+    const result = await execa(`git`, [`config`, `user.name`])
+    if (result.failed) {
+      delete packageJson.author
+    } else {
+      packageJson.author = result.stdout
+    }
+  } catch (e) {
+    delete packageJson.author
+  }
+
+  await fs.writeJSON(packageJsonPath, packageJson)
```

Comment:
Does it prettify by default? `JSON.stringify(obj, null, 2)`?

Reply:
We don't need to prettify because we do it immediately before running npm install, which does its own formatting